python_modules/libraries/dagster-pandas/dagster_pandas/data_frame.py

```diff
@@ -87,6 +87,16 @@ def df_type_check(_, value):
         success=True,
         metadata_entries=[
             EventMetadataEntry.text(str(len(value)), 'row_count', 'Number of rows in DataFrame'),
+            EventMetadataEntry.text(str(value.shape), "shape", "The shape of the DataFrame."),
+            EventMetadataEntry.md(
+                value.dtypes.to_markdown(), "dtypes", "The dtypes within the DataFrame."
+            ),
+            EventMetadataEntry.md(
+                dataframe.head().to_markdown(), "head", "A preview of the first 5 rows."
```

Comment:
I don't think this works. The dataframe object is encapsulated in the value parameter. This might be the root of the failing checks.

Reply:
@asingh16 hmmm, that's strange. It definitely works on our deployment!

Reply:
@asingh16 oh oh I see what you mean now. Forgot to rename `dataframe` to `value`. Fixing!